Dockerfile.test.php7.debug

```diff
@@ -1,12 +1,12 @@
-FROM php:7.2
+FROM php:8.0.0
 
 RUN apt-get update && \
     apt-get install -y mariadb-client zlib1g-dev
 
-RUN yes | pecl install xdebug-2.8.0
+RUN yes | pecl install xdebug-3.0.0
 RUN echo "zend_extension=$(find /usr/local/lib/php/extensions/ -name xdebug.so)" > /usr/local/etc/php/conf.d/xdebug.ini
-RUN echo "xdebug.remote_enable=on" >> /usr/local/etc/php/conf.d/xdebug.ini
-RUN echo "xdebug.remote_autostart=on" >> /usr/local/etc/php/conf.d/xdebug.ini
+RUN echo "xdebug.mode=debug" >> /usr/local/etc/php/conf.d/xdebug.ini
```

Comment:
@kongtiaowang I believe xdebug >= v3.0.0 supports php 8 and the settings for xdebug.remote_enable and xdebug.remote_autostart have been renamed. See https://xdebug.org/docs/upgrade_guide#changed-xdebug.remote_enable